batch/batch/worker/worker.py

```diff
@@ -352,6 +364,9 @@ def free(self, netns: NetworkNamespace):
 
     async def _free(self, netns: NetworkNamespace):
         await netns.cleanup()
+        await netns.create_netns()
+        await netns.enable_iptables_forwarding()
+        await netns.mark_packets()
```

Comment:
These setup steps are duplicated in `init`. Let's pull them out into a separate function so it's not as easy to include a step in one place and not the other. I had initially tried to leave the iptables forwarding rules in place so we're not constantly deleting and recreating the same rules, but I might be misguided that that was a worthwhile performance shortcut. Do you need to delete and recreate the mangle rules for correctness in the bookkeeping?

Reply:
No - it's just a counter and we look at the difference between two time points rather than the absolute number. I added that because I thought it was a change you requested previously. Let me know what you prefer.

Reply:
Sorry for the confusion, I wanted to make sure that there was no garbage that would grow unbounded, and forgot about what I had done with the other iptables rules. Regardless though, leaving these rules in would not grow unbounded since we reuse the same veths and only need to create the rules once in the init. Given that we face all this contention with iptables, let's not add repeated work and just do the packet-marking once in `init`. Sorry!

Reply:
ok. I reverted everything back to the old behavior and we just add the new mangle rules on init now.